lib/index.js

```diff
@@ -508,7 +513,11 @@ function _openDatabase(options, callback) {
         };
       }
       // authSource should be set in connectOptions
-      const opts = {...config.authentication, ...config.connectOptions};
+      const opts = {
+        ...config.authentication,
+        ...config.connectOptions,
+        writeConcern
```

Comment:
```suggestion
        ...config.writeOptions.writeConcern
```

Reply:
this would result result in the options inside of writeConcern being spread into the options itself. So I just decided to do

```js
...config.writeOptions
```

here: https://github.com/digitalbazaar/bedrock-mongodb/pull/65/commits/163a951857899ac70ee2944a892eb1b6eda2a484